Sasquatch/Sasquatch/ViewControllers/MSDocumentDetailsViewController.swift

```diff
@@ -3,19 +3,214 @@
+    }
+    return .none
+  }
+
+  func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCellEditingStyle, forRowAt indexPath: IndexPath) {
+    if shouldDisplayProperties(in: indexPath.section) {
+      userDocumentAddPropertiesSection.tableView(tableView, commit: editingStyle, forRowAt: indexPath)
+    }
+  }
+
+  func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+    tableView.deselectRow(at: indexPath, animated: true)
+    if shouldDisplayProperties(in: indexPath.section) && userDocumentAddPropertiesSection.isInsertRow(indexPath) {
+      self.tableView(tableView, commit: .insert, forRowAt: indexPath)
+    }
+  }
+
+  func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+    if shouldDisplayProperties(in: indexPath.section) {
+      return userDocumentAddPropertiesSection.tableView(tableView, cellForRowAt:indexPath)
+    } else if documentContent != nil {
+      let cell = tableView.dequeueReusableCell(withIdentifier: "property", for: indexPath)
+      var cellText = ""
+      switch indexPath.row {
+        case 0:
+          cellText = "Document ID: \(documentContent?.documentId ?? "")"
+        break
+        case 1:
+          cellText = "Partition: \(documentContent?.partition ?? "")"
+        break
+        case 2:
+          guard let error = documentContent?.error else {
+            let formatter = DateFormatter()
+            formatter.dateFormat = "dd MMM yyyy HH:mm"
+            cellText = "Date: \(formatter.string(from: documentContent?.lastUpdatedDate ?? Date()))"
+            break
+          }
+          cellText = "Error: \(error.error.localizedDescription)"
+        break
+        case 3:
+          guard (documentContent?.error) != nil else {
+            cell.textLabel?.numberOfLines = 0;
+            cell.textLabel?.lineBreakMode = NSLineBreakMode.byWordWrapping;
+            let dictionary = documentContent?.deserializedValue.serializeToDictionary() ?? [:]
+            do {
+              let jsonData = try String(data: JSONSerialization.data(withJSONObject: dictionary, options: .prettyPrinted), encoding: String.Encoding.utf8)
+              cellText = "Document content: \(jsonData ?? "unknown")"
+            } catch {
+              cellText = "Document content could not be deserialized."
+            }
+            break
+          }
+        break
+      default:
+        cellText = "nil"
+      }
+      cell.textLabel?.text = cellText
+      return cell
+    } else {
+      let cell = tableView.dequeueReusableCell(withIdentifier: "property", for: indexPath)
+      return cell
+    }
+  }
+  
+  func convertTimeToLiveConstantToValue(_ constValue : String) -> Int {
+    switch constValue {
+      case TimeToLiveMode.Infinite.rawValue:
+      return -1
+      case TimeToLiveMode.NoCache.rawValue:
+      return 0
+      case TimeToLiveMode.TwoSeconds.rawValue:
+      return 2000
```

Comment:
Unit is in seconds, not milliseconds.
```suggestion
      return 2
```